package.json

```diff
@@ -27,7 +37,19 @@
     "uuid": "^3.0.1"
   },
   "devDependencies": {
```

Comment:
Not exactly sure why it's needed and overcomplicate things? Correct me if I'm wrong. 🐸